### ISSUE TITLE
Return 400 rather than 500 from API endpoints when params missing 

### ISSUE THREAD

**Issue by agnessa** [Vizzuality/trase#348]
So far we handled missing parameters in API using a `500` status with a `{error: '...'}` payload. I believe it would be more appropriate to use `400` in such cases, but perhaps the choice of `500` had to do with easier handling on the frontend. Happy to discuss.

**Comment by sorodrigo** [comment]
I'm not aware of anything depending on error statusCode. If @tiagojsag doesn't know something I don't you can go ahead and change it.

**Comment by tsubik** [comment]
I definitely agree that 400 is more suitable here :+1:

**Comment by tiagojsag** [comment]
@sorodrigo no, AFAIK the client only cares about success vs failure, it doesn't care much who's to blame. Let's move ahead with this change
